app/_src/gateway/reference/expressions-language/language-references.md

```diff
@@ -165,8 +177,8 @@ Here are the allowed combination of field types and constant types with each ope
 |----------------------------------|-----------------------------------------|----------------|----------|----------------------------------|---------|--------------|
 | `String`                         | `==`, `!=`, `~`, `^=`, `=^`, `contains` | ❌              | ❌        | ❌                                | `~`     | ❌            |
 | `IpAddr`                         | ❌                                       | `in`, `not in` | `==`     | ❌                                | ❌       | ❌            |
-| `Int`                            | ❌                                       | ❌              | ❌        | `==`, `!=`, `>=`, `>`, `<=`, `<` | ❌       | ❌            |
-| `Expression`                     | ❌                                       | ❌              | ❌        | ❌                                | ❌       | `&&`, `      |
+| `Int`                            | ❌                                       | ❌              | ❌        | `==`, `!=`, `>=`, `>`, `<=`, `<` | ❌       | ❌           |
+| `Expression`                     | ❌                                       | ❌              | ❌        | ❌                                | ❌       | `&&`, `||`  |
```

Comment:
I think this one needs a conditional rendering tag too.

Reply:
@dndx Is this addition of `||` specific to 3.6? I just noticed that there's a comma after the `&` in 3.5, so maybe this is change applies to older versions as well?

Reply:
Hmm. Based on https://github.com/Kong/docs.konghq.com/pull/6824/files#diff-3c6d333d12a566458ed0dda376674205ea702272cb3427bed7c37f873b809bc7R142-R148, it should be `||` in 3.5 and earlier, and `\|\|` in 3.6 onward.

Reply:
@Guaris @lena-larionova I added the conditional rendering, but for some reason it's showing a blank space before `Expressions` in 3.6: https://deploy-preview-6824--kongdocs.netlify.app/gateway/dev/reference/expressions-language/language-references/#type-and-operator-semantics And I don't know why since I used another one on the page as an example.

Reply:
Instead of a whole new row, try using the `inline` flag: https://docs.konghq.com/contributing/conditional-rendering/#inline-content

I also wonder if the renderer is reading `||` as a table cell or something. This might work:
`{% raw %}||{% endraw %}`

Just surround the above with backticks in the markdown as well.

Reply:
@lena-larionova That worked, thanks!